hack/run_test

```diff
@@ -308,21 +294,28 @@ function run_replication_test() {
       sleep 1
     done
   done
+}
+
+function test_value_replication() {
+  local table_name="$1"; shift
 
+  # Setup the replication data
   local value
   value=24
-  # Do some real work to test replication in practice
-  postgresql_cmd -c "CREATE TABLE t1 (a integer); INSERT INTO t1 VALUES ($value);"
+  local table_name="t1"
+  postgresql_cmd -c "CREATE TABLE $table_name (a integer); INSERT INTO $table_name VALUES ($value);"
 
   # Read value from slaves and check whether it is expected
   local f
   local slave_ip
+  local cidfile
+  local master_ip=$(get_container_ip master-$cid_suffix.cid)
   for f in `find ${CIDFILE_DIR} -type f -name "slave-*"`; do
     cidfile=$(basename $f)
     slave_ip=$(get_container_ip $cidfile)
     CONTAINER_IP=$slave_ip
     for i in $(seq $max_attempts); do
-      result="$(postgresql_cmd -At -c "select * from t1")"
+      result="$(postgresql_cmd -At -c "select * from $table_name")"
       if [[ ! -z "${result}" ]]; then
```

Comment:
This is probably OK, but I would rather fix it sooner than later ... checking psql output 
for (non)empty-ness is not perfect, because error messages comes to standard output ->
thus empty string comes from invalid query.  We are fine because 'set -e' is activated ..
but that could change in future...  I would vote for explicit string to compare $result with.